content/hi/docs/reference/glossary/cluster-operator.md

```diff
@@ -0,0 +1,22 @@
+---
+title: क्लस्टर ऑपरेटर
+id: cluster-operator
+date: 2018-04-12
+full_link:
+short_description: >
+  एक व्यक्ति जो क्लस्टर को कॉन्फ़िगर, नियंत्रित और मॉनिटर करता है।
+
+aka:
+tags:
+  - user-type
+---
+
+एक व्यक्ति जो क्लस्टर को कॉन्फ़िगर, नियंत्रित और मॉनिटर करता है।
+
+<!--more-->
+
+उनकी प्राथमिक जिम्मेदारी एक क्लस्टर को ऊपर रखना और चलाना है, जिसमें आवधिक रखरखाव गतिविधियां या उन्नयन शामिल हो सकते हैं।<br>
```

Comment:
Rather than ऊपर रखना, wouldn't चालू रखना sound more appropriate?

Reply:
Very nice suggestion
@divya-mohan0209 I have changed into all your suggested

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good